features/k8s-service-credential-distribution.md

```diff
@@ -0,0 +1,221 @@
+
+**API**
+
+- [ ] Initial API review.
+
+**Approvals**
+
+- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [ ] The TOC has reviewed and approved promotion of the feature as part of the
+	roadmap for a release.
+
+**Promotion**
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+
+---
+
+## Beta
+
+### Requirements: 
+
+**Design**
+
+- [ ] Design doc describing the intention of the feature, how it will be
+	implemented, and any thoughts on how to test the feature has been approved by
+	relevant work group leads
+- [ ] Feature coverage and test plans written and approved.
+
+**Docs** 
+
+- [ ] Documentation on istio.io includes performance expectations; may have caveats. 
+- [ ] Documentation on istio.io includes samples/tutorials. 
+- [ ] Documentation on istio.io includes appropriate glossary entries. 
+- [ ] All new documentation containing user actions includes istio.io tests.
+- [ ] Release notes have been added. 
+- [ ] Upgrade notes have been added. 
+
+**Tests**
+
+- [ ] Integration tests cover feature edge cases
+- [ ] End-to-end tests cover samples/tutorials
+- [ ] Fixed issues have tests to prevent regressions
+- [ ] Stability/stress test suite includes coverage for the feature.
+
+**Performance**
+
+- [ ] Feature coverage and test plans written and approved 
+- [ ] Tests exist with the feature enabled that can be integrated with our automated performance testing.
+
+**API**
+
+- [ ] TOC has reviewed the API and determined it to be complete. 
+
+**Tooling**
+
+- [ ] Any necessary tooling to use/debug the feature has been implemented and is complete. 
+
+**Bugs**
+
+- [ ] Feature has no known major issues.
+
+**Approvals**
+
+- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [ ] The supportability review panel has reviewed promotion of the feature.  
+- [ ] The TOC has reviewed and approved promotion of the feature as part of the
+	road map for a release.
+
+
+**Promotion**
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Stable
+
+### Requirements: 
+
+**Performance**
+
+- [ ] Latency, throughput, and scalability are quantified and documented on
+	istio.io. 
+
+**Bugs**
+
+- [ ] Feature has no known major issues. 
+
+**Approvals**
+
+- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [ ] The [supportability review panel](https://docs.google.com/document/d/1w0epyFhhDSf_TwFEfa_lrn1v61mXNJKpEp_kUgp4sSc/edit#) has reviewed the feature in order to find any supportability concerns.  
+- [ ] The TOC has reviewed and approved promotion of the feature as part of the
```

Comment:
Check for 1 and 3. Do we need supportability review for existing stable features?